calblueprint/ogsc
# Add Admin Notes to Scheme and UI

# Adding Admin Notes to User Invites

I added a column in the users table called admin notes, which is displayed when an admin accesses an invite. 

## TODO (if applicable)

n/a

## How to Test/Use PR feature

You can add something in the adminNotes column of the database, and see the reflected changes in each invite. 

## PR Checklist

Does your branch (check if true):
[x] work when you run `yarn build`?
[x] successfully run `yarn:db-migrate up` without yielding any errors?
[x] successfully run `yarn prisma introspect` without yielding any errors?
[x] successfully run `yarn dev` and support all changes that your branch intends to perform?

## Related PRs

n/a

## Migrations

added AdminNotes table

## Screenshots

<img width="769" alt="Screen Shot 2021-02-25 at 10 50 21 PM" src="https://user-images.githubusercontent.com/56060477/109265746-e5a02700-77bb-11eb-96fe-3b6002a7924d.png">

CC: @sydbui
